spec/helpers/application_helper_spec.rb

```diff
@@ -25,4 +25,27 @@
       end
     end
   end
+
+  describe "title" do
+
+    context "nil input" do
+      it 'returns nil' do
+        expect(title).to be nil
+      end
+    end
+
+    context "non-nil input" do
```

Comment:
for context descriptions its conventional to use the format "**when** input is nil" etc